custom-fields/array-of-guids/array-of-guids-readonly.component.html

```diff
@@ -0,0 +1 @@
+<input class="custom-title-input" [value]="value" readonly />
```

Comment:
do we actually need this template? Can we just reuse the write component template and set a readonly prop to the settigns obj

Reply:
because it is just a sample of how to add a custom extension for this type of field. I guess it is more expected the implementations for write and readonly components to be different from one another in a real scenario.